test/trivy_ignore_test.go

```diff
@@ -0,0 +1,393 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aquasecurity/defsec/pkg/terraform"
+
+	"github.com/aquasecurity/defsec/pkg/severity"
+
+	"github.com/aquasecurity/defsec/pkg/scan"
+
+	"github.com/aquasecurity/defsec/internal/rules"
+
+	"github.com/aquasecurity/defsec/pkg/providers"
+
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
lint:

```suggestion
	"fmt"
	"testing"

	"github.com/aquasecurity/defsec/internal/rules"
	"github.com/aquasecurity/defsec/pkg/providers"
	"github.com/aquasecurity/defsec/pkg/scan"
	"github.com/aquasecurity/defsec/pkg/severity"
	"github.com/aquasecurity/defsec/pkg/terraform"
	"github.com/stretchr/testify/assert"
)
```

If you're IntelliJ, you can setup `gofmt` to run as a file watcher ever time you make a change.